src/Nav.js

```diff
@@ -113,24 +116,25 @@ Nav.propTypes = {
    * HTML id for the inner `ul` element
    */
   ulId: React.PropTypes.string,
-  expanded: React.PropTypes.bool,
   navbar: React.PropTypes.bool,
   eventKey: React.PropTypes.any,
   pullRight: React.PropTypes.bool,
-  right: React.PropTypes.bool
+  right: deprecated(React.PropTypes.bool, 'Use the `pullRight` prop instead')
```

Comment:
Do we need to expose `pullRight` as a special prop on `<Nav>` at all? Maybe just have something like `navbarRight`? Or only support `right` when used with `navbar` and fail on prop type validation otherwise? Users can always add `.pull-right` themselves if they need.